bin/alluxio-start.sh

```diff
@@ -43,11 +43,12 @@ MOPT (Mount Option) is one of:
              set ALLUXIO_RAM_FOLDER=/dev/shm on each worker and use NoMount.
   SudoMount is assumed if MOPT is not specified.
 
--f  format Journal, UnderFS Data and Workers Folder on master
--N  do not try to kill previous running processes before starting new ones
--w  wait for processes to end before returning
--m  launch monitor process to ensure the target processes come up.
--h  display this help."
+-f         format Journal, UnderFS Data and Workers Folder on master.
+-h         display this help.
+-i backup  a journal backup to restore the master from.
```

Comment:
can you mention the format of this backup, and/or examples?

Reply:
good idea, done